pkg/reconciler/instances/rma/integration_action.go

```diff
@@ -362,12 +369,13 @@ func findLatestRevision(releases []*release.Release) *release.Release {
 func generateVmalertGroup(context *service.ActionContext, id, num string) int {
 	groups, err := strconv.Atoi(num)
 	if err != nil {
-		context.Logger.Debugf("got error %s when converting string to int for configuration: %s, use its default value: %d", err, RmiVmalertGroupsNum, DefaultVMAlertGroupsNum)
+		context.Logger.Debugf("got error %s when converting string to int for configuration: %s, use its default value: %d",
+			err, RmiVmalertGroupsNum, DefaultVMAlertGroupsNum)
 		groups = DefaultVMAlertGroupsNum
 	}
 
 	csum := sha256.Sum256([]byte(id))
-	mrand.Seed(int64(binary.LittleEndian.Uint64(csum[0:8])))
+	mrand.New(mrand.NewSource(int64(binary.LittleEndian.Uint64(csum[0:8]))))
```

Comment:
Same problem with `math/rand` - the returned object is not used, it looks like this has no effect.